src/impl/vamp/random/uniform.hh

```diff
@@ -7,9 +7,9 @@
 
 namespace vamp::rng
 {
-    struct RNG
+    struct Random
```

Comment:
I feel like this type (a) shouldn't be defined in a header specific to uniform-random sampling and (b) should be called something like `Distribution` instead. 

Also, if this type exists, the various planners should instantiate it instead of using the RNG directly, to be more explicit about the kind of samples they are drawing.